custom/icds/templates/icds/messaging/indicators/tam/dpt3_and_measles_due.txt

```diff
@@ -0,0 +1,3 @@
+{# NOTE: Django assumes templates are stored with UTF-8 encoding by default. Be sure to save this template with UTF-8 encoding. #}
+
+{{ child_name }} को इस महीने डी. पी. टी.-३ और खसरे   का टीका लगाना हैं पर अभी तक उसे यह टीका नहीं लगे हैं |
```

Comment:
Hindi?

Reply:
Yes, Hindi. This indicator was never released, so I didn't add Tamil translations.